alacritty_terminal/src/term/mod.rs

```diff
@@ -418,12 +426,17 @@ impl<T> Term<T> {
             }
 
             if !cell.flags.intersects(Flags::WIDE_CHAR_SPACER | Flags::LEADING_WIDE_CHAR_SPACER) {
+                if esc_seqs {
+                    cell.as_escape(buf, last_cell);
```

Comment:
I'm wondering if it wouldn't makes sense to split up this PR into one for the basic functionality and then a follow-on for adding back in various escapes. Personally, I think this should be two separate commands users can choose between anyways, much like how copy/paste today work without this. If done properly, perhaps we could think about adding a new kind of `Copy` action as well.

Reply:
With escape sequences things might get slowed down significantly though, so an approach that works for plain text might not be workable for colored text.

Reply:
A lot of the code is going to be the same. Spawning the process, modifying the config, etc.

Reply:
> A lot of the code is going to be the same. Spawning the process, modifying the config, etc.

That is mostly true as long as we keep it asynchronous.

Reply:
It is done